### PR TITLE
Convert read_timestamp to event.created in Journalbeat

### PR DESCRIPTION
**Pull request by ruflin**
This aligns journalbeat with ECS.

### CODE REVIEW

**Comment by ruflin** [comment]
@kvch Are there other things need in journalbeat that need alignment with ECS?

**Comment by kvch** [comment]
I think `host.name` needs to be renamed to `host.hostname`. Do you want to include that in this PR?

**Comment by ruflin** [comment]
Trying also to convert it directly in this PR. Can you check if I picked the correct field?

File: journalbeat/reader/fields.go [hunk 1]
```diff
@@ -62,7 +62,7 @@ var (
 		sdjournal.SD_JOURNAL_FIELD_COMM:              fieldConversion{"process.name", false, false},
 		sdjournal.SD_JOURNAL_FIELD_EXE:               fieldConversion{"process.executable", false, false},
 		sdjournal.SD_JOURNAL_FIELD_GID:               fieldConversion{"process.uid", true, false},
-		sdjournal.SD_JOURNAL_FIELD_HOSTNAME:          fieldConversion{"host.name", false, false},
+		sdjournal.SD_JOURNAL_FIELD_HOSTNAME:          fieldConversion{"host.hostname", false, false},
```
Comment (kvch) [inline journalbeat/reader/fields.go]: :+1:

**Comment by kvch** [comment]
I don't see anything else changed recently.

I am curious if you are planning to add `facility` and `identifier` fields in some form to ECS? In journalbeat there are a few syslog fields. I recalled having those in ECS. :)

**Comment by ruflin** [comment]
@kvch +1 on making these change too. Could you open a PR for it?

**Comment by ruflin** [comment]
Hm, tests seem to fail because of the changes. I hoped my adjustment works, seems like I need to create a setup. @kvch If you have some hints to get this green, let me know.

**Comment by kvch** [comment]
The problem is that an empty object is still there with the key `"event"`.
```
Actual: {"event":{},"host":{"boot_id":"123456"}}
Expected: {"host":{"boot_id":"123456"}}
```
The whole key-value "pair" has to be deleted:
```golang
event.Fields.Delete("event")
```

**Pull request merged**